aexpect/client.py

```diff
@@ -1093,7 +1093,7 @@ def cmd_output_safe(self, cmd, timeout=60):
                 o += self.read_up_to_prompt(0.5)
                 success = True
                 break
-            except ExpectError, e:
+            except ExpectError as e:
```

Comment:
IIRC this is not compatible with python2.4. I know we officially don't support it in avocado, but we still need to support it in avocado-vt, right?